extract.js

```diff
@@ -0,0 +1,16 @@
+import {Project} from 'ts-morph';
+
+const project = new Project({
+	tsConfigFilePath: 'tsconfig.json',
+});
+const testSourceFiles = project.getSourceFile('index.ts');
+for (const tag of testSourceFiles.getSymbol('isRepo').getExport('isRepo').getJsDocTags()) {
```

Comment:
Given `testSourceFiles.getSymbol('isRepo').getExport('isRepo')`, I can't reach the FunctionDeclaration to then call `addJsDoc` 🤷‍♂️ 

I knew function expressions were a mistake, I think `testSourceFiles.getExportDeclaration('isRepo')` would have gotten me there.